src/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -311,9 +313,11 @@ public async Task<ActionResult> Delete(Guid instanceGuid, int instanceOwnerParty
                 InstanceOwnerPartyId = instance.InstanceOwner.PartyId,
                 User = new PlatformUser
                 {
-                    UserId = User.GetUserIdAsInt(),
+                    UserId = User.GetUserId(),
                     AuthenticationLevel = User.GetAuthenticationLevel(),
```

Comment:
Should we do anything about authentication level for system users? altinn-authentication adds authlevel 3 for Maskinporten token exchange

Reply:
What do you have in mind? Authentication level is already handled as far as I know.

Reply:
`GetAuthenticationLevel` will return `0`/`null` here since the claim doesn't exist, since systemuser tokens aren't exchanged to Altinn tokens so there is no `authlevel` claim. But technically both a normal service owner token and a system user token is based on the same authentication mechanism (Maskinporten), so it seems strange that the authentication level isn't the same. I'm not sure if the property is used for anything?